ports/atmel-samd/README.rst

```diff
@@ -1,17 +1,8 @@
-SAMD21x18
-=========
+SAMD21 and SAMD51
+==================
 
-This port brings MicroPython to SAMD21x18 based development boards under the name
-CircuitPython. Supported boards include:
-
--  Adafruit CircuitPlayground Express
--  Adafruit Feather M0 Basic
--  Adafruit Feather M0 Express
--  Adafruit Metro M0 Express
--  Adafruit M0 Bluefruit LE
--  Arduino Zero
--  Arduino MKR Zero
--  Arduino Nano 33 IoT
+This port supports many development boards that utilize SAMD21 and SAMD51 chips. See
+https://circuitpython.org/downloads for all supported boards.
 
 
 Pinout
```

Comment:
As long as you are editing this, I'd suggest just deleting this Pinout section, which only includes a few boards and can't easily be kept up to date.